src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -604,6 +613,9 @@ export function DashboardTopNav({
                   data-test-subj="dashboardAddPanelButton"
                 />
               ),
+              addControlButton: showToolbarButton
+                ? dashboardAppState.dashboardContainer.controlGroup?.getControlButton('toolbar')
```

Comment:
Note that, if the new controls are not enabled in Labs, then `controlGroup` won't be defined - therefore, this toolbar button *won't* be rendered even if `showToolbarButton` is somehow true! No extra checks necessary - cool! 👍 (This is more of a note to myself since I was confused at first, LOL 😄)